tests/ctst/world/Zenko.ts

```diff
@@ -197,36 +224,39 @@ export default class Zenko extends World {
      * @returns {undefined}
      */
     async prepareARWWI(ARWWIName: string, ARWWIPassword: string, ARWWITargetRole: string) {
+
         if (!(ARWWIName in CacheHelper.ARWWI)) {
-            const token = await this.getWebIdentityToken(
+            const token: string = await this.getWebIdentityToken(
                 ARWWIName,
                 ARWWIPassword,
-                this.parameters.KeycloakHost || 'keycloak.zenko.local',
-                this.parameters.keycloakPort || 80,
-                `/auth/realms/${this.parameters.keycloakRealm || "zenko"}/protocol/openid-connect/token`,
-                this.parameters.keycloakClientId || Constants.K_CLIENT,
-                this.parameters.keycloakGrantType || 'password',
+                this.parameters.KeycloakHost as string || 'keycloak.zenko.local',
+                this.parameters.keycloakPort as string || '80',
+                `/auth/realms/${this.parameters.keycloakRealm as string || 'zenko'}/protocol/openid-connect/token`,
+                this.parameters.keycloakClientId as string || Constants.K_CLIENT,
+                this.parameters.keycloakGrantType as string || 'password',
             );
             this.options.webIdentityToken = token;
             if (!this.options.webIdentityToken) {
                 throw new Error('Error when trying to get a WebIdentity token.');
             }
             // Getting account ID
             const account = await SuperAdmin.getAccount({
-                name: this.parameters.AccountName || Constants.ACCOUNT_NAME,
-            });
+                name: this.parameters.AccountName as string || Constants.ACCOUNT_NAME,
+            }) as Utils.AccountObject;
             // Getting roles with GetRolesForWebIdentity
-            const { data: { data: roles } } = await SuperAdmin.getRolesForWebIdentity(this.options.webIdentityToken);            // Get the first role with the storage-manager-role name
-            let roleToAssume = '';
-            if (roles.ListOfRoleArns) {
-                roleToAssume = roles.find(
-                    (roleArn: string) => roleArn.includes(ARWWITargetRole) && roleArn.includes(account.id),
+            // Get the first role with the storage-manager-role name
+            const data: GetRolesForWIResponse =
+                (await SuperAdmin.getRolesForWebIdentity(this.options.webIdentityToken)).data as GetRolesForWIResponse;
+            let roleToAssume: string | undefined = '';
+            if (data.roles.ListOfRoleArns) {
+                roleToAssume = data.roles.ListOfRoleArns.find(
+                    (roleArn: string) => roleArn.includes(ARWWITargetRole) && roleArn.includes(account.id as string),
```

Comment:
If the `account` object is already of the right type, the `as string` is not needed)

Reply:
The problem is the account object has a type of 
```
AccountObject {
    id?: string;
    name?: string;
    arn?: string;
}
```
so id can be undefined, so must be cast here. Do you suggest in CTST making these variables non optional?

Reply:
See my comment from the latest review about the use of `!`